### PR TITLE
web: remove const qualifier for getMaterialInstances

### PR DESCRIPTION
**Pull request by mackong**
With the *const* qualifier on the returned MaterialInstances, we cannot call setter functions like *setDepthWrite* etc. from the object, caused by
```
BindingError: Cannot convert argument of type MaterialInstance const* to parameter type MaterialInstance*
```

### CODE REVIEW

File: web/filament-js/filament.d.ts [hunk 1]
```diff
@@ -624,7 +624,7 @@ export class gltfio$FilamentInstance {
     public getSkinNames(): Vector<string>;
     public attachSkin(skinIndex: number, entity: Entity): void;
     public detachSkin(skinIndex: number, entity: Entity): void;
-    public getMaterialInstances(): Vector<MaterialInstance>;
```
Comment (romainguy) [inline web/filament-js/filament.d.ts:627]: Why this change here? The other getters return a Vector<>.

Comment (mackong) [reply web/filament-js/filament.d.ts:627]: *getMaterialVariantNames* is wrapper by *vectorToArray*,  which is convenient when iterate, so make this change to *getMaterialInstances* too.
What's the convention (vectorToArray or not) here, should all interfaces defined in *filament.d.ts* use Vector?

Comment (bejado) [reply web/filament-js/filament.d.ts:627]: I think we should keep this consistent with the other APIs and continue to return a `Vector<MaterialInstance>`.

**Review by bejado — CHANGES_REQUESTED** [review]

File: web/filament-js/filament.d.ts (hunk 1 above)
Comment (mackong) [reply web/filament-js/filament.d.ts]: Sure, reverted.
